frontend/public/components/RBAC/bindings.jsx

```diff
@@ -287,15 +287,16 @@ const rowFilters = [
   },
 ];
 
-export const RoleBindingsPage = ({
+const RoleBindingsPage_ = ({
   namespace = undefined,
   showTitle = true,
   mock = false,
   staticFilters = undefined,
   createPath = '/k8s/cluster/rolebindings/~new',
+  flags,
 }) => (
   <MultiListPage
-    canCreate={true}
+    canCreate={!flags[FLAGS.SHOW_OPENSHIFT_START_GUIDE]}
```

Comment:
Just use `mock`

```suggestion
    canCreate={!mock}
```

Reply:
Ok, updaetd to `!mock` although I don't understand what `mock/fake` is or how it relates to Getting Started?

Reply:
`mock` basically says don't show a real result. It's the disabled empty state. We have a higher order component that sets `mock` to true when `SHOW_OPENSHIFT_START_GUIDE` (but we'd want to disable the button however `mock` got set)